.github/workflows/ci.yml

```
@@ -11,22 +11,20 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - uses: purescript-contrib/setup-purescript@main
-
       - uses: actions/setup-node@v1
         with:
-          node-version: "12"
+          node-version: "18"
 
       - name: Install dependencies
         run: |
-          npm install -g bower
-          npm install
-          bower install --production
+          sudo apt install chezscheme
+
+      - name: Setup PureScript dependencies
+        run: npm i --global purescript@0.15.10 purs-tidy@latest spago@next purescript-psa@latest purescm@next
```

Spago includes PSA now:

```suggestion
        run: npm i --global purescript@0.15.10 purs-tidy@latest spago@next purescm@next
```